### ISSUE TITLE
Errata management guide link broken

### ISSUE THREAD

**Issue by lzap** [theforeman/foreman-documentation#115]
```
URL        `https://access.redhat.com/documentation/en-us/red_hat_satellite/6.7-beta/html/errata_management_guide/'
Name       `Errata Management'
Parent URL file:///home/lzap/work/foreman-documentation/guides/build/doc-Installing_Foreman_on_Debian/index-foreman.html, line 3305, col 4
Real URL   https://access.redhat.com/documentation/en-us/red_hat_satellite/6.7-beta/html/errata_management_guide/
Check time 3.376 seconds
Result     Error: 404 Not Found
```

We should not be merging any PR which has red validation tests. Master is currently broken, there are more of these.

**Comment by lzap** [comment]
One more:

```
URL        `https://access.redhat.com/documentation/en-us/red_hat_satellite/6.7-beta/html/package_manifest/'
Name       `Package Manifest'
Parent URL file:///home/lzap/work/foreman-documentation/guides/build/doc-Installing_Satellite_Server_Connected/index-foreman.html, line 2411, col 570
Real URL   https://access.redhat.com/documentation/en-us/red_hat_satellite/6.7-beta/html/package_manifest/
Check time 2.469 seconds
Result     Error: 404 Not Found
```

**Comment by melcorr** [comment]
@lzap I updated the Product version to 6.7-beta so that some of the current links that did not exist in 6.6 would be fine, but these two links are not present for the beta, only in 6.6. So, if I go back to 6.6, I will break other links. This was the problem I mentioned on our call a few weeks ago, and this will be a recurring issue as we come close to releasing every time. 

We have not been merging anything that fails to pass the automated tests. You merge half the PRs and the others on my team merge the others. They've all been green.

**Comment by melcorr** [comment]
Attribute to fix this is part of PR #114

**Comment by melcorr** [comment]
This is resolved as part of #114
